web/thesauruses/python/control_structures.json

```diff
@@ -0,0 +1,103 @@
+{
+  "meta": {
+    "language": "language_id",
+    "language_version": "version.number",
+    "language_name": "Human-Friendly Language Name"
+  },
+  "categories": {
+    "Conditional Statements (ifs)": [
+      "if_conditional",
+      "if_else_conditional",
+      "if_elseif_conditional",
+      "if_elseif_else_conditional"
+    ],
+    "Loops": [
+      "while_loop",
+      "do_while_loop",
+      "until_loop",
+      "do_until_loop",
+      "for_loops",
+      "foreach_loops"
+    ],
+    "Iterations": [
+      "each_iteration",
+      "map_iteration",
+      "filter_iteration",
+      "fold_iteration",
+      "recursion_iteration"
+    ]
+  },
+  "control_structures": {
+    "if_conditional": {
+      "name": "If conditional",
+      "code": "
+      if condition:
+        statements"
+    },
+    "if_else_conditional": {
+      "name": "If/Else conditional",
+      "code": "
+      if condition:
+        statements
+      else:
+        statements"
+    },
+    "if_elseif_conditional": {
+      "name": "If/ElseIf conditional",
+      "code": "
+      if condition:
+        statements
+      elif:
+        statements"
+    },
+    "if_elseif_else_conditional": {
+      "name": "If/ElseIf/Else conditional",
+      "code": "
+      if condition:
+        statements
+      elif condition:
+        statements"
+    },
+    "while_loop": {
+      "name": "While loop",
+      "code": "
+      while condition:
+        statements"
+    },
+    "do_while_loop": {
+      "name": "Do/While loop",
+      "code": "Unsupported or Not Needed"
```

Comment:
For these, if you put this in quotes it will show up as code. You can just delete "name" and "code" out of this do_while_loop block. (same for the others below)

Reply:
Removed the code and name